src/rebuild/scan.c

```diff
@@ -887,14 +903,16 @@ rebuild_container_scan_cb(daos_handle_t ih, vos_iter_entry_t *entry,
 			 rebuild_obj_scan_cb, NULL, arg, dth);
 	dtx_end(dth, NULL, rc);
 
+	*acts |= VOS_ITER_CB_YIELD;
```

Comment:
This is less efficient than not setting it.   vos_iterate does a check to determine if there was something else scheduled during the callback.   Setting this will force it to re-probe even if it's not needed because the yield was a noop.  The user should never set this flag.  Perhaps I should remove it from the interface altogether.